extensions/2.0/Khronos/KHR_materials_diffuse_transmission/README.md

```diff
@@ -0,0 +1,241 @@
+<img src="figures/thin_translucent.jpg" height="350" />
+</p>
+<sub><figcaption><em>(Thin) translucent leaves/foliage</em></figcaption></sub>
+</figure>
+
+## Extending Materials
+
+The effect is activated by adding the `KHR_materials_diffuse_transmission` extension to any glTF material.
+
+```json
+{
+    "materials": [
+        {
+            "extensions": {
+                "KHR_materials_diffuse_transmission": {
+                    "diffuseTransmissionFactor": 0.25,
+                    "diffuseTransmissionTexture": {
+                      "index": 0
+                    },
+                    "diffuseTransmissionColorFactor": [
+                      1.0,
+                      0.9,
+                      0.85
+                    ]
+                }
+            }
+        }
+    ]
+}
+```
+
+|                                     | Type                                                                | Description                                                                                                                                                                                         | Required                 |
+| ----------------------------------- | ------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------ |
+| **diffuseTransmissionFactor**       | `number`                                                            | The percentage of reflected, non-specularly reflected light that is transmitted through the surface via the Lambertian diffuse transmission, i.e., the strength of the diffuse transmission effect. | No, default: `0`         |
+| **diffuseTransmissionTexture**      | [`textureInfo`](/specification/2.0/README.md#reference-textureInfo) | A texture that defines the strength of the diffuse transmission effect, stored in the alpha (A) channel. Will be multiplied by the diffuseTransmissionFactor.                                       | No                       |
+| **diffuseTransmissionColorFactor**  | `number[3]`                                                         | The color of the transmitted light.                                                                                                                                                                 | No, default: `[1, 1, 1]` |
+| **diffuseTransmissionColorTexture** | [`textureInfo`](/specification/2.0/README.md#reference-textureInfo) | A texture that defines the color of the transmitted light, stored in the RGB channels and encoded in sRGB. This texture will be multiplied by diffuseTransmissionColorFactor.                       | No                       |
+
+
+The strength of the effect is controlled by `diffuseTransmissionFactor` and `diffuseTransmissionTexture`, combined via multiplication to describe a single value.
```

Comment:
I think that a more suitable wording would be along the lines of:
"The percentage of light that is transmitted through the surface is calculated by multiplication of `diffuseTransmissionFactor` and `diffuseTransmissionTexture` to describe a single value"

Since there is not really any 'strength' :-)

Reply:
Recently the OpenPBR effort appears to have adopted the term "weight" where glTF has traditionally used the word "strength."  I guess I like "weight" better, but we should check our existing ratified extensions to see how hard it will be to make the change.  Anisotropy, for example, uses "strength" normatively, and I'm sure others do as well.  We should figure out how important it is to have consistency on this term among glTF extensions.

Reply:
> I'm sure others do as well

Occlusion uses strength also in the core spec. Not that I'm advocating using this term here.

Reply:
I really think we must approach this from a physical point of view:
The parameter governs the percentage (factor) of light that is transmitted into the surface - to be re-emitted at the other side.

From the side being illuminated - this could be seen as absorbed or internally transmitted factor.
From the side facing away from the light - this could be seen as the re-emitted  factor.

I fully understand that editors/dcc tools call this something akin to 'weight' or 'strength' because that is understandable to content creators - in the glTF spec I think we need to be more accurate and use 'percentage' or 'amount' instead.

Reply:
Changing from "strength" to some other terminology is a much larger discussion, one that retroactively touches the glTF core specification and many ratified PBR extensions.  I don't think this PR is the correct venue to propose such a change.

Reply:
Really?
I don't understand that reasoning.
Just because we use a technical wording in this extension does not mean that we have to make changes to existing spec.

Reply:
Just from a consistency point of view, I wouldn't want to use the term "strength" in one extension and then "weight" in another extension, and then have to explain why we used two different terms to mean the same thing.